config/locales/es.yml

```diff
@@ -170,3 +170,4 @@ es:
   editor: "Editor"
   console: "Consola"
   type_expressions_here: "Acá podés escribir expresiones para probar tu solución"
+  repeat_pending: "Terminar pendientes"
```

Comment:
Ok, the different color and caption are representative enough, and the
arrow fits in better.

El jue., sept. 10, 2015 14:34, Franco Leonardo Bulgarelli <
notifications@github.com> escribió:

> In config/locales/es.yml
> <https://github.com/mumuki/mumuki-platform/pull/303#discussion_r39188724>:
>
> > @@ -170,3 +170,4 @@ es:
> >    editor: "Editor"
> >    console: "Consola"
> >    type_expressions_here: "Acá podés escribir expresiones para probar tu solución"
> > +  repeat_pending: "Terminar pendientes"
>
> Ok, maybe you are right.
>
> We are using here repeat
> <http://fortawesome.github.io/Font-Awesome/icon/repeat/> icon. What about
> simply using next icon
> <http://fortawesome.github.io/Font-Awesome/icon/chevron-right/> like the
> rest?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mumuki/mumuki-platform/pull/303/files#r39188724>.
>